tracker/tracker_debugger_api.yaml

```
@@ -61,6 +61,41 @@ paths:
             example: john@doe.com
           in: query
           required: false
+        - name: page_url
+          description: Filter. A part of page URL of the event, that have to be in the session to display the session
+          schema:
+            type: string
+            example: blog
+          in: query
+          required: false
+        - name: event_type
+          description: Filter. A type of the event, that have to be in the session to display the session
+          schema:
+            type: int
```

Are you sure int is a valid type? https://swagger.io/docs/specification/data-models/data-types/#numbers
```suggestion
            type: integer
```